### PR TITLE
Direct JS lib link & inform about Git submodules

### PR DESCRIPTION
**Pull request by WietseWind**
## High Level Overview of Change

Update Readme to contain links to the JS SDK as well (since someone using this JS lib will probably prefer to see the JS parent lib, instead of an overview with SDK's for other languages.

And: add a note about having to init and update Git submodules first, as other `npm run test` (and all other commands) will fail anyway.

### Context of Change

Dev clones, runs build/test/... » fails. Checks individual commands, checks the .sh and notices missing directories. Then checks the directories and finds out it requires a submodule.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,30 @@
...
+Xpring Common JavaScript Library provides common JavaScript functionality to all client side libraries in [Xpring SDK](https://github.com/xpring-eng/xpring-sdk) ([TS/JS version](https://github.com/xpring-eng/xpring-js)). This library is built to be consumed as a dependency to other libraries, rather than a standalone library.
 
-Developers probably want to use one of the language specific libraries for [Xpring SDK](https://github.com/xpring-eng/xpring-sdk#client-side-libraries), rather than consuming this library directly. However, developers with specific requirements for interacting with Xpring SDK may want to depend on this library.
+Developers probably want to use one of the language specific libraries for [Xpring SDK](https://github.com/xpring-eng/xpring-sdk#client-side-libraries) ([TS/JS version](https://github.com/xpring-eng/xpring-js)), rather than consuming this library directly. However, developers with specific requirements for interacting with Xpring SDK may want to depend on this library.
+
+## Getting started
```
Comment (keefertaylor) [inline README.md]: There's a bit in [CONTRIBUTING.md](https://github.com/xpring-eng/xpring-common-js/blob/master/CONTRIBUTING.md#building-the-library) about how to build the library. However, you're not the first person to have this problem, so our current model of where to find the build instructions is clearly wrong. 

Would you consider either:
1) Linking that section here
2) Removing the section from CONTRIBUTING.md and putting it here

I'll propagate whatever changes you make to this file structure to the other Xpring SDK libraries as well.

**Review by keefertaylor — APPROVED** [review]
LG - sorry for latency as it was a long weekend in the US.

**Comment by keefertaylor** [comment]
Adding a second reviewer @WietseWind. Feel free to merge as soon as the GitHub UI lets you.

**Review by 0xASK — APPROVED** [review]

**Comment by WietseWind** [comment]
@keefertaylor Any idea when `ci/circleci: build Expected — Waiting for status to be reported` will run?

**Comment by keefertaylor** [comment]
Sorry this PR got lost because it didn't show up in my queue since it's approved. 

I think that we had a setting toggled wrong in CircleCI to not build pull requests. Could you merge master into this branch and see if that kicks off a run?

**Comment by WietseWind** [comment]
@keefertaylor That worked. However: `Only those with write access to this repository can merge pull requests.`

image

**Comment by keefertaylor** [comment]
Ah, apologies. Still working on grok'ing the GitHub and associated tools permissions. Merging now.

**Pull request merged**